CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## master / unreleased
-
+* [ENHANCEMENT] rulers: Add support to persist tokens in rulers. #5987
```

Comment:
Please move the changelog entry to `ENHANCEMENT` section.

From https://github.com/cortexproject/cortex/blob/master/RELEASE.md, changelog needs to be ordered by

```
Ensure changelog entries for the new release are in this order:
[CHANGE]
[FEATURE]
[ENHANCEMENT]
[BUGFIX]
```

Reply:
Done, thanks for pointing out.